staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -127,73 +125,6 @@ func makeValidPrometheusMetricLabel(in string) string {
 // call chains to NewReflector, so they'd be low entropy names for reflectors
 var internalPackages = []string{"client-go/tools/cache/", "/runtime/asm_"}
```

Comment:
can we have `"/runtime/asm_"` somewhere centrally and only append to that locally?

Reply:
> can we have "/runtime/asm_" somewhere centrally and only append to that locally?

done